app/components/admin/edition_images/uploaded_images_component.rb

```diff
@@ -4,4 +4,37 @@ class Admin::EditionImages::UploadedImagesComponent < ViewComponent::Base
   def initialize(edition:)
     @edition = edition
   end
+
+  def lead_image
+    if @edition.first_image_pulled_out? && !@edition.images.empty?
+      image_to_object @edition.images.first, 0
+    end
+  end
+
+  def document_images
+    lead_image_count = @edition.first_image_pulled_out? ? 1 : 0
+    if @edition.images.length > lead_image_count
```

Comment:
can this just be if @edition.includes(Edition::FirstImagePulledOut) && @edition.images.any?